docs/technical-appendices/i18n.md

```diff
@@ -2,12 +2,12 @@
 
 ## Locales implemented and translation status
 
-<!-- DO NOT EDIT! (START fr|1803|0|0|en|1803|1700|0) -->
+<!-- DO NOT EDIT! (START en|1802|1699|0|fr|1802|0|0) -->
 | Locale | Messages | Empty | Empty mandatory | PO file | Translators |
 |--|:--:|:--:|:--:|:--:|:--:|
-|<img src="../../src/web/static/images/locales/fr.svg" style="height: 1em;"/>&nbsp;``fr``&nbsp;Français | 1803 | 0 | 0 | [messages.po](../locale/fr/LC_MESSAGES/messages.po) | [Pascal AUBRY](https://github.com/pascalaubry)<br/>[Sammy PLAT](https://github.com/Amaras) |
-|<img src="../../src/web/static/images/locales/en.svg" style="height: 1em;"/>&nbsp;``en``&nbsp;English | 1803 | 1700 | 0 | [messages.po](../locale/en/LC_MESSAGES/messages.po) | [Timothy ARMES](https://github.com/timothyarmes) |
-Last update: 2025-08-01 14:57
+|<img src="../../src/web/static/images/locales/en.svg" style="height: 1em;"/>&nbsp;``en``&nbsp;English | 1802 | 1699 | 0 | [messages.po](../locale/en/LC_MESSAGES/messages.po) | [Timothy ARMES](https://github.com/timothyarmes) |

```

Comment:
It's starting to get really annoying to see the format so unstable (switching between French first and English first).
Why does it do that?